src/components/Header/index.tsx

```diff
@@ -55,7 +56,7 @@ export default function Header() {
     toggleDarkModeAnalytics(!darkMode)
     toggleDarkModeAux()
   }, [toggleDarkModeAux, darkMode])
-  const swapState = useSwapTradeState()
+  const swapRawState = useSwapRawState()
   const { state: tradeState } = useTradeState()
```

Comment:
I know this is not new, but what's the difference between tradeState and swapTradeState?
From the name alone I would guess there's overlap between them

Reply:
`tradeState` could be a state of Swap or Limit orders or Advanced orders. `swapRawState` is exactly state of Swap. Here `swapRawState` is used as a fallback